docs/content/dagster-cloud/getting-started/getting-started-with-serverless-deployment.mdx

```diff
@@ -0,0 +1,137 @@
+
+---
+
+## Step 2: Select Serverless as your deployment type
+
+Serverless deployment is a fully-managed version of Dagster Cloud that allows you to run your Dagster jobs without spinning up any infrastructure. While Serverless is the easiest way to get started with Dagster, this deployment type works best with jobs that aren't computationally expensive. Refer to the [Serverless guide](/dagster-cloud/deployment/serverless) for more info.
+
+If you find that Serverless isn't right for you, you can switch to a Hybrid deployment at any time.
+
+**To use this deployment type**, click **Select** under **Serverless deployment** and confirm when prompted.
+
+---
+
+## Step 3: Deploy your code
+
+Now that you've set up your deployment, you'll need to tell it the location of your Dagster code.
+
+A code location specifies a single Python package or file that defines your Dagster code. Each Dagster Cloud deployment can include code from one or more code locations. Refer to the [Adding a code location guide](/dagster-cloud/developing-testing/code-locations) for more info.
+
+<details>
+	<summary><strong>USING GITHUB</strong></summary>
+
+1. Click **Choose account or organization** and sign into your GitHub account.
+2. Select the account or organization where Dagster should be installed.
+3. After the installation and authorization is complete, you'll be redirected back to Dagster Cloud.
+4. After Dagster Cloud is connected, you'll be prompted to either:
+
+   - **Clone the Dagster Cloud starter project** - This option clones the [dagster-cloud-template](https://github.com/dagster-io/dagster-cloud-template) repository.
+
+     1. Enter a **Repository name**.
+     2. Select a **Repository visibility**.
+     3. Click **Clone and deploy.**
+
+   - **Use an existing repo** - This option uses an existing repository in your GitHub account.
+     1. Select a **repository**.
+     2. Click **Deploy**.
+
+</details>
+
+<details>
+	<summary><strong>USING THE DAGSTER-CLOUD CLI</strong></summary>
+
+To complete this step using the CLI, you can use your own Dagster code or the [`dagster-cloud-examples` code](https://github.com/dagster-io/dagster-cloud/tree/main/dagster-cloud-examples). Our examples use the `dagster-cloud-examples` package.
+
+1. To install the `dagster-cloud` CLI and the `dagster-cloud-examples` package, run:
+
+   ```shell
+   pip install dagster-cloud dagster-cloud-examples
+   ```
+
+2. Complete the authentication step, including [creating a user token](/dagster-cloud/account/managing-user-agent-tokens#managing-user-tokens).
+
+3. Run the following to deploy your code to Dagster Cloud:
+
+   ```shell
+   dagster-cloud serverless deploy "<PATH_TO_CODE_DIRECTORY>" \
```

Comment:
I'm not sure the formatting here is correct - confirm y/n, please

Reply:
First, log in to Dagster Cloud via the CLI

```
dagster-cloud configure
```

Next deploy your code

```
dagster-cloud serverless deploy \
  --location-name <LOCATION_NAME>
  --package-name <PACKAGE NAME>
```